### PR TITLE
Fixes issues with Alerts and Status

### PR DESCRIPTION
**Pull request by bipuladh**
Screenshot from 2023-04-06 17-39-37

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
@bipuladh LGTM
can u squash the commits ?
also, there is a lint warning (introduced by one of my PRs) for `TopologyNavigation.tsx`, can u plz pass the controller to the dependency list as well ?

**Comment by SanjalKatiyar** [comment]
LGTM
cc @alfonsomthd (for further reviews)

File: packages/shared/topology/utils/AlertFilters.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  };
+export const nodeFilter = (name: string) => (alert: Alert) =>
+  alert?.labels?.node === name || commonFilter(name)(alert);
+
+export const filterRelevantAlerts = (
+  resourceName: string,
+  resourceModel: K8sModel,
+  alerts: Alert[]
+): Alert[] => {
+  if (!resourceName || !!alerts) return [];
```
Comment (alfonsomthd) [inline packages/shared/topology/utils/AlertFilters.ts:21]: Shouldn't we check is there are no alerts?
```suggestion
  if (!resourceName || _.isEmpty(alerts)) return [];
```

Comment (bipuladh) [reply packages/shared/topology/utils/AlertFilters.ts:21]: ack

**Pull request closed without merging**